src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Match.cs

```diff
@@ -397,9 +397,10 @@ internal override void Dump()
         {
             if (_caps != null)
             {
-                foreach (DictionaryEntry kvp in _caps)
+                foreach (object? entry in _caps)
                 {
-                    System.Diagnostics.Debug.WriteLine("Slot " + kvp.Key.ToString() + " -> " + kvp.Value.ToString());
+                    DictionaryEntry kvp = (DictionaryEntry)entry!;
```

Comment:
I'd appreciate feedback on how to do this more cleanly.

Reply:
This is fine.  It's also debug-only code.